client/js/screens/StoryTrail/StoryTrail.js

```diff
@@ -14,96 +14,198 @@ import AudioItem from '../../components/AudioItem';
 import styles from './styles';
 import {THEME} from '../../config/';
 
-const StoryTrail = () => {
-  return (
-    <ScrollView style={styles.container}>
-      <Image
-        style={styles.imageMainBackground}
-        source={{
-          uri:
-            'https://bcparksfoundation.ca/site/assets/files/1313/storytrail_header_web_3.1920x0.jpg',
-        }}
-        resizeMode="cover"
-      />
-      <View style={styles.wrapperTrailImage}>
-        <ImageBackground
-          style={styles.circleTrailImage}
-          source={require('../../assets/images/imgBgSelfGuidedTourAudio.png')}>
-          <Image
-            style={styles.TrailImage}
-            resizeMode="cover"
+class StoryTrail extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      audioPlayer: null,
+      paused: true,
+      currentTime: 0,
+      duration: null,
+      draggingSlider: false,
+      draggingTime: 0,
+    };
+  }
+
+  getMinutes = time => {
+    if (time) {
+      let minutes = Math.floor(time / 60);
+      minutes = minutes < 10 ? `0${minutes}` : minutes;
+      return minutes;
+    } else {
+      return 0;
+    }
+  };
+  getSeconds = time => {
+    if (time) {
+      let seconds = Math.round(time % 60);
+      seconds = seconds < 10 ? `0${seconds}` : seconds;
+      return seconds;
+    } else {
+      return 0;
+    }
+  };
```

Comment:
what is 10 and 60? Please use const to store values, otherwise, they are kist magic numbers.

Reply:
It is used to change the time which type is `string` to be like `01` when it is smaller than `10`